iOS Client/PodcastsController/PodcastsCoordinator.swift

```diff
@@ -43,10 +48,45 @@ class PodcastsCoordinator: Coordinator, StoreSubscriber {
     
     // MARK: - StoreSubscriber
     func newState(state: MBAppState) {
+        
+        switch state.podcastsState.player {
+        case .initialized:
+            break
+        case .playing:
+            if let podcast = state.podcastsState.selectedPodcast, podcast.guid != currentPlayingPodcast?.guid {
+                start(podcast)
+                currentPlayingPodcast = state.podcastsState.selectedPodcast
+            } else if case .paused = playerState {
+                player.play()
+            }
+        case .paused:
+            player.pause()
+        case .error:
+            break
+        }
+        
+        playerState = state.podcastsState.player
+        
         guard state.navigationState.selectedTab == .podcasts, let newRoute = state.navigationState.routes[.podcasts] else {
             return
         }
         build(newRoute: newRoute)
         route = newRoute
     }
+    
+    func start(_ podcast: MBPodcast) {
+        if let guid = podcast.guid,
+        let url = URL(string:guid) {
+            let item = AVPlayerItem(url: url)
+            player.replaceCurrentItem(with: item)
+            player.play()
+        }
+    }
+}
+
+enum PlayerState {
```

Comment:
do this belong here or in `MBState` ?

Reply:
I can put it in MBState, it's probably more appropriate there